.clang-tidy

```diff
@@ -28,4 +28,25 @@ Checks:          '-*,
                   '
 #WarningsAsErrors: '*'
 AnalyzeTemporaryDtors: false
+# Use Google style guide.
+CheckOptions:
+# Classes / Structs / Enums are CamelCase.
+  - { key: readability-identifier-naming.ClassCase,               value: CamelCase  }
+  - { key: readability-identifier-naming.StructCase,              value: CamelCase  }
+  - { key: readability-identifier-naming.EnumCase,                value: CamelCase  }
+# Variables / Parameters are lower_case.
+  - { key: readability-identifier-naming.VariableCase,            value: lower_case }
+  - { key: readability-identifier-naming.ParameterCase,           value: lower_case }
+  - { key: readability-identifier-naming.PublicMemberCase,        value: lower_case }
```

Comment:
Forgot to mention, but we need a `PrivateMemberCase` and `PrivateMemberPrefix` (or suffix if that's what we decide to use).

Reply:
ah that's true.

Reply:
Fixed.